views/layouts/main.php

```diff
@@ -143,6 +143,18 @@
 //                            ]
                             //\SILEX:info
                         ]];
+        $menuItems[] = ['label' => Yii::t('app', 'Data Analysis'), 
```

Comment:
Maybe you should add a config parameter for the data analysis part. If a platform don't have this module, they should not have the link in the menu.

Reply:
good idea

Reply:
Nice I add it.